### ISSUE TITLE
AMD option targeting developers, content creators, and power users

### ISSUE THREAD

**Issue by cmack** [system76/laptop-suggestions#14]
### Why/User Benefit/User Problem
It's hard to find laptop makers that pair premium features with AMD CPUs. For example, models with high resolution screens and large memory and storage capacity are not usually provided with AMD CPU-based SKUs. Or they are only available with "gaming" segment models that are usually bulkier and heavier.

The high-core-count AMD CPUs paired with premium features would appeal to developers and content creators. 

### Description of the feature
1. Provide an option for latest generation AMD CPUs 
2. Give the same high grade options that other SKUs using different CPU manufacturers receive. For example, if the other SKU's highest grade config allows for 4K monitor, 1TB storage, and 16+ GB of RAM, the AMD SKU should have those options too.

**Comment by kolgza** [comment]
What would be really interesting is if the [AMD PSP ](https://en.wikipedia.org/wiki/AMD_Platform_Security_Processor)could be disabled the same way the [Intel Management Engine was](https://liliputing.com/2017/11/system76-will-disable-intel-management-engine-linux-laptops.html).

One can only dream.

**Comment by AlNotSteve** [comment]
Up vote this!

**Comment by rakshith-ravi** [comment]
An AMD system, with a small form factor (13-14 inch), comfortable, no-numpad keyboard (like the ThinkPads) with a high-end GPU (enough to game 60+ fps with an external monitor) would be the most ideal system of all time. Can we please make this happen @system76?

**Comment by barrel0luck** [comment]
Yes! Yes! Yes! Given all the security nightmares with Intel chips, AMD option will be much appreciated!

**Comment by mtkunkel** [comment]
Is AMD + Thunderbolt 3 do-able yet?   Potentially a first in the market..   (one can dream)

**Comment by victorolinasc** [comment]
Fully performant GPU with mainline kernel only...

No graphics driver blob...

No kernel update hell... 

I dare you System 76.... I double dare you...

**Comment by Owslla** [comment]
Up voting this suggestion.
